src/subgraph/index.ts

```diff
@@ -1,4 +1,57 @@
+import { getAllOrdersByUserAddress, getAllPendingOrders, getOrderById, getPythPriceIdsForOrderIds } from './orders';
+import { Chain } from '../common/chain';
+import { PythConfig, RelayerConfig, RpcConfig, SubgraphConfig } from '../types';
+import { getAllPositionsByUserAddress, getPositionById } from './positions';
+import { getAllMarketsFromSubgraph, getMarketById } from './markets';
+
 export * from './common';
 export * from './markets';
 export * from './orders';
 export * from './positions';
+
+export class SubGraph {
+  private rpcConfig: RpcConfig;
+  private subGraphConfig: SubgraphConfig;
+
+  constructor(rpcConfig: RpcConfig, subGraphConfig: SubgraphConfig) {
+    this.rpcConfig = rpcConfig;
+    this.subGraphConfig = subGraphConfig;
+  }
+
+  // orders
+  public async getAllOrdersByUserAddress(userAddress: string) {
```

Comment:
you should add the `count` and `skip` fields to allow pagination other way you will only see always the first 10 request. They should be optional to allow user query all request if are not presents this params